qa/smoke-test-ingest-with-all-dependencies/src/test/java/org/elasticsearch/ingest/IngestDocumentMustacheIT.java

```diff
@@ -31,7 +31,7 @@
 
 import static org.hamcrest.Matchers.equalTo;
 
-public class IngestDocumentMustacheIT extends AbstractMustacheTests {
+public class IngestDocumentMustacheIT extends AbstractMustacheTestCase {
```

Comment:
Why is this an IT when it indirectly extends ESTestCase? Why do we have java code running at all in what should be a rest test??